app/assets/javascripts/helpers/interestingHelper.js

```diff
@@ -31,61 +33,77 @@ define([], function() {
     },
     stay_informed: {
       title: 'Keep Updated',
-      description: 'Learn what is currently happening in forests on the ground, read more about data, sign up to receive updates, and learn how the GFW platform continues to evolve to meet user needs',
+      description: 'Learn what is currently happening in forests on the ground, read more about data, sign up to receive updates, and learn how the GFW platform continues to evolve to meet user needs.',
       link: '/stayinformed',
     },
     how_to: {
       title: 'Need help?',
-      description: 'Learn how to get the most out of the Global Forest Watch platform',
+      description: 'Learn how to get the most out of the GFW platform.',
       link: '/howto',
     },
     about_GFW: {
       title: 'About GFW',
-      description: 'Learn more about the GFW platform',
+      description: 'Learn more about the GFW platform.',
       link: '/about',
     },
     about_fires: {
       title: 'About GFW Fires',
-      description: 'Learn about a new tool to monitor emerging fires, identify potential causes, and analyze impacts of fires on forests and people.',
+      description: 'Learn about GFW Fires, a tool to monitor emerging fires, identify potential causes, and analyze impacts of fires on forests and people.',
       link: 'http://fires.globalforestwatch.org/#v=about&x=115&y=0&l=5&lyrs=Active_Fires',
     },
     about_commodities: {
       title: 'About GFW Commodities',
-      description: 'Learn more about the GFW platform',
+      description: 'Learn more about the GFW Commodities, a tool to address deforestation in commodity supply chains.',
       link: 'http://commodities.globalforestwatch.org/#v=about&n=GFW',
     },
     faqs: {
       title: 'FAQs',
-      description: 'Read through answers to the most frequently asked questions about GFW',
+      description: 'Read through answers to the most frequently asked questions about GFW.',
       link: '/howto/faqs',
     },
     provide_feedback: {
       title: 'Provide feedback',
-      description: 'Help us improve the website by submitting feedback or by signing up to become an official GFW tester',
+      description: 'Help improve GFW by submitting feedback or by signing up to become an official GFW tester.',
       link: '/getinvolved/provide-feedback',
+      button_text: 'Submit feedback',
     },
     contact_us: {
       title: 'Contact us',
-      description: 'Contact the Global Forest Watch team with questions, comments, and feedback',
+      description: 'Contact the GFW team with questions, comments, and feedback.',
       link: '/about/contact-us',
+      button_text: 'Contact us',
+    },
+    explore_applications: {
+      title: 'Explore GFW applications',
+      description: 'Explore GFW apps, tools designed to address specific forest issues.',
+      link: '/explore/applications',
+      button_text: 'Explore apps',
+    },
+      coming_soon: {
+      title: 'Coming soon to GFW,
```

Comment:
This is no nice! But it lacks the end simple quote mark in this line :) We'll merge this after adding it!